Modules/_decimal/_decimal.c

```diff
@@ -3569,30 +3619,35 @@ pydec_format(PyObject *dec, PyObject *context, PyObject *fmt, decimal_state *sta
 }
 
 /* Formatted representation of a PyDecObject. */
+
+/*[clinic input]
+_decimal.Decimal.__format__
+
+    self as dec: self
+    fmtarg: object
+    override: object(c_default="NULL") = None
```

Comment:
```suggestion
    format_spec as fmtarg: object
    override: object = NULL
    /
```
They are positional-only, and `override` can only be a dict.

Reply:
> They are positional-only

M'kay.  Maybe it's better to revert this?  It's not required by AC transition.

> override can only be a dict.

I know.  Did T miss some AC-magic, which can remove PyDict_Check?!

Reply:
What revert? These parameters are positional-only, so you mast use `/` in Argument Clinic.

`None` is not valid default value for `override`.